editors/sc-ide/widgets/code_editor/editor.cpp

```diff
@@ -855,12 +874,21 @@ void GenericCodeEditor::resetFontSize() { mDoc->resetDefaultFont(); }
 void GenericCodeEditor::zoomFont(int steps) {
     QFont currentFont = mDoc->defaultFont();
     const int newSize = currentFont.pointSize() + steps;
-    if (newSize <= 0)
+    if (newSize <= 7 || newSize >= 50)
```

Comment:
The pinch zoom works great now - if "feels" good to me and the range is good as well.

However, I'm unsure about this particular change. Right now with the default settings on macOS I can get a slightly smaller font using pinch zoom out than with `cmd -`.

I'd either remove this change, keeping the old behavior in place (no limit), or clamp it to the same value as pinch zoom - `std::clamp(newSize, defaultFontSize * 0.5, defaultFontSize * 8.0);`.

Reply:
Oh, I somehow oversaw that, thanks for pointing this out. I added a `GenericCodeEditor::clampFontSize` function via https://github.com/supercollider/supercollider/pull/6563/commits/caca3c6f7ca54f3ae737cc91ad585fcc3ce89b88 which is also used by the Post Window and also by the keyboard zoom.